### PR TITLE
Raise exception when the files in BIGDL_JRAS does not exist

### PR DESCRIPTION
**Pull request by yangw1234**
This PR improves the error message when the files specified in BIGDL_JRAS does not exists.

This error is not uncommon when developing using IDE.

### CODE REVIEW

**Comment by yangw1234** [comment]
@hkvision Could you help review it?

File: pyzoo/zoo/util/engine.py [hunk 1]
```diff
@@ -107,6 +107,9 @@ def get_analytics_zoo_classpath():
     Get and return the jar path for analytics-zoo if exists.
     """
     if os.getenv("BIGDL_CLASSPATH"):
+        for path in os.getenv("BIGDL_CLASSPATH").split(":"):
+            if not os.path.exists(path):
+                raise ValueError("Path {} specified BIGDL_CLASSPATH does not exists.".format(path))
```
Comment (hkvision) [inline pyzoo/zoo/util/engine.py:112]: exists -> exist

Comment (yangw1234) [reply pyzoo/zoo/util/engine.py]: fixed

**Comment by hkvision** [comment]
LGTM

**Comment by yangw1234** [comment]
http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/3237/ jenkins pass, example tests failed due to hdfs issue

**Pull request merged**